jupyter-spark/extensions/spark.js

```diff
@@ -1,4 +1,5 @@
-var API = "/spark/api/v1";
+var BASE_URL = Jupyter.utils.get_body_data("baseUrl") || "/";
```

Comment:
I think this should be `base-url` rather than `baseUrl` - when I view the source on a running notebook locally, I see an attribute on the `body` tag for `data-base-url`.